packages/uniorg-parse/src/reader.ts

```diff
@@ -1,3 +1,4 @@
+import { Point, Position } from 'unist';
```

Comment:
minor:
```suggestion
import type { Point, Position } from 'unist';
```
so `unist` can remain a dev dependency